Menu/Provider/GroupMenuProvider.php

```diff
@@ -147,9 +147,9 @@ public function get($name, array $options = array())
                 if (isset($item['admin']) && !empty($item['admin'])) {
                     $admin = $this->pool->getInstance($item['admin']);
 
-                    // skip menu item if no `list` url is available or user doesn't have the LIST access rights
+                    // Return null if no `list` url is available or user doesn't have the LIST access rights
                     if (!$admin->hasRoute('list') || !$admin->hasAccess('list')) {
-                        continue;
+                        return null;
```

Comment:
Link for the lazy : https://github.com/sonata-project/SonataAdminBundle/blob/master/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php#L105